packages/token-bridge-sdk/src/hooks/useBalance.ts

```diff
@@ -9,7 +14,7 @@ type Erc20Balances = {
   [address: string]: BigNumber | undefined
 }
 
-const merge: Middleware = useSWRNext => {
+const merge: Middleware = (useSWRNext: SWRHook) => {
```

Comment:
I'm wondering if we can make use of this to preserve old data instead of middleware
https://swr.vercel.app/blog/swr-v2#preserving-previous-state
but i'm not sure, would love to get @chrstph-dvx 's input
though it doesn't necessarily have to be done on this PR

Reply:
I don't think so. `merge` is only about merging data, so if you initially have `[1,2]` and fetch `[3]`, then you would have `[1,2,3]`. I think `keepPreviousData` would return previous data (`[1,2]`) but once merging is done, it would return `[3]`